docs/source/_templates/page.html

```diff
@@ -56,8 +68,7 @@
     });
 </script>
 
-{% endblock %}
-
-{% block regular_scripts %}{{ super() }}
+{% endblock %} {% block regular_scripts %}{{ super() }}
 <script async defer src="https://buttons.github.io/buttons.js"></script>
+<script src="https://doc755kimnz5y.cloudfront.net/latest/agility-web-component.js"></script>
```

Comment:
can we update this to `https://cdn.cleanlab.ai/latest/agility-web-component.js`. Got the new domain set up last night!

Reply:
amazing